Add Starter kit, Glossary & Kickstart your community page (#1189)
galaxyproject/galaxy-hub: content/get-started/index.md

---
title: Get Started
---

> <a href="https://galaxy-mentor-network.netlify.app/" class="btn btn-primary" role="button" aria-pressed="true">Would you like to learn about Galaxy with a mentor? Check out the Galaxy Mentorship Network</a>

Reviewer: I am not sure I would add this link here (I mean without context). As a newcomer, I don't think one would "jump" into a mentoring program without first knowing about the community itself and what it does.

Reviewer: The mentorship program is mostly useful when you need guidance on a project or need help learning how to contribute to open source and for settling into the Galaxy ecosystem.

So I guess you need to "navigate" a bit the Galaxy ecosystem; at least you went through some of the Galaxy Tutorials; maybe you asked for help in the Galaxy help forum and then realise you need a bit more than this "punctual" help. So I would put it after the help forum (but is is definitely open to discussion!!!).

Author: I'd add just what you described as an introduction for it (or smth similar)

 _You went through some of the Galaxy Tutorials; maybe you asked for help in the Galaxy help forum and then realise you need a bit more than this "punctual" help. What about reaching to the Galaxy Mentorship Network?_

## Welcome!

> Below you will find tutorials, useful links and definitions as the starting point for newcomers,students, experts and anybody who wants to refresh their knowledge about Galaxy's basics and capabilities.

Reviewer: space missing: newcomers,students --> newcomers, students

Author: Fixed, thx

### Galaxy Glossary

> [List](link) of all keywords used within Galaxy and their definitions.

Reviewer: It should point to the glossary, right?

I would also introduce a bit the "why" for the glossary. Maybe something like (and sorry I am not very good at making sentences...):

*The Galaxy glossary contains all the terminology, acronyms and their definitions. As a newbee, you will probably skip many of its content but later on in your Galaxy journey, you will come back to it and it will help you to get a common understanding of the Galaxy ecosystem.*

Author: Yes it should! 

I like the sentence, maybe 

_The Galaxy glossary contains all the terminology, acronyms and their definitions. As a newbie, you will probably skim through this list but later on in your Galaxy journey you will probably come back to refer to it as it will help you get a better understanding of the Galaxy Ecosystem._

### Set up your community 

> Do you happen to not find your community within Galaxy? Are there specific requirements you'd like to have? If you'd like to create your add yours as part of the Galaxy Communities we have a dedicated page on how to start, see [here](https://galaxyproject.org/new-leads

Reviewer: I would put first the tutorials.

|Introduction to the GTN |[<i class="fab fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/slides/introduction.html#1
){:target="_blank"} |   | [<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=lDqWxzWNk1k){:target="_blank"} |   |
| How to contribute using Gitpod |   | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/gitpod/tutorial.html){:target="_blank"}  | [<i class="fas fa-video" aria-hidden="true"></i>](https://youtu.be/_-wnsPttBCo){:target="_blank"}  |   |
|How to create a new tutorial in the GTN   |   |[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html){:target="_blank"}   |[<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=Vw9h5kvRH2k){:target="_blank"}  |   |   |

> 

Reviewer: I would put a link here about the Galaxy Community. And before setting up your community, link to regional communities, scientific communities. and only if you don't find anything that suit you you could consider setting up your community.

And I would probably change the title to something a bit mroe "soft". "What if you can't find yourself in these lists of regional and/or scientific communities?)

### Set up your community

Author: Okay,agreed!  Two separate titles it is then

> ### Useful links

Reviewer: This section is difficult to "read" because it does not seem to have a clear "purpose".

Author: My idea was to put in one place links useful maybe not the first time someone joins Galaxy but later on as they get to know it and get more curious and discover them. 

 An introduction to the list of useful links may be better

- [FAQs quickstart](https://training.galaxyproject.org/training-material/faq)
  
  Feel like you might have a question that many others have had as well? Check our Frequently Asked Questions quickstart!

- [Public Galaxy Servers](https://galaxyproject.org/use/)
  

>   List of all Public Galaxy Servers available to use or deploy your own version.

Reviewer: I would put it before Tutorials. To follow a tutorial you need to understand that there are different servers. 
We could explain this and tell newcomers that they will have to "choose" a server (and provide guidance depending on what they are willing to do, etc.).

Author: Okay, after reading this comment maybe it'd be better to divide the links under "Useful links" into different "items" like this two?
So it's not as confusing?

- [Galaxy Training Network](https://training.galaxyproject.org/)
  
  Collection of tutorials from a variety of topics() developed and maintained by the worldwide Galaxy community.

> 

Reviewer: I would probably put this after your table with link to the tutorials you have chosen to start.

- [Galaxy Help](https://help.galaxyproject.org/)
  

>   Have a bug or an error you haven't been able to solve? Do you have a question that you haven't found in already answered in the FAQs? Reach out to the Galaxy Help Community.

Reviewer: I would also probably put this after the galaxy server and before tutorial (not sure). Because when you start a tutorial, you want to know where you can ask for help if you get stuck.

- [Galaxy Communities](https://galaxyproject.org/community/)
  

>   List and description of already stablished communities.

Reviewer: I would put it before the section about Set up your community. We need to know what exists before starting anything new.

- [Galaxy Publication Library](https://www.zotero.org/groups/1732893/galaxy)
  

>   Publication library for the Galaxy Project that use, reference, implement, and extend Galaxy.

Reviewer: Why do we need to add this link here? Is it to help newbees to find existing papers where Galaxy is used? I think we would need to add a bit more background information; otherwise remove it.

Author: I was conflicted were to add this or not, I found it deep somewhere in Galaxy and asked Bea if it'd be nice to have it. She was cool with it as there'd be some that  could be curious about papers related to the Galaxy Community.

More background how so?

- [Events](https://galaxyproject.org/events/)
  
  Upcoming and past events with Galaxy-related content.

> 

Reviewer: Should'nt we have it earlier? At least before starting a tutorial on your own you may want to check any upcoming training event.

- [Mailing lists](https://galaxyproject.org/mailing-lists/)

>   

Reviewer: Mailing lists aim at complementing the Galaxy Help Forum so it could be "close" to the help forum item.

  Links, email address and archives for Galaxy's Mailing Lists, you can reach out to the community in this format, ask your questions and get to know what's up with Galaxy.

- [Working groups](https://galaxyproject.org/community/wg/)

>   

Reviewer: I would also add a bit more background information. Why a newbee would be interested by working groups e.g. at which stage of his Galaxy journey.  I thin kit is more when you think about "contributing" to Galaxy so it could be in a "Ready to contribute"  section.

  List of current Working Groups set within the Galaxy Ecosystem. 

- [Galaxy's repositories](https://github.com/galaxyproject)
  

>   Galaxy Project Repositories including the GTN,Planemo, Galaxy and The Hub.

Reviewer: Same as previous comment. Useful when you think to contribute and/or deploy.

Author: Repo's are linked under the "how to contribute" page. This link is under the same idea overall under "Useful links"

### Want to contribute?

> [Here's](https://galaxyproject.org/community/contributing/) our guide on how to start contributing and there are many ways to do so: coding,writing and creating tutorials, answering questions in Gitter or the Help Forum and many more. 

Reviewer: I would put it before the previous points.

Author: Agreed, will change

### Bug reporting etiquette

> We have the Help Forum to ask away your questions. If you still are unsure, you can always go to our open issues in GitHub to see if someone reported it, reach out to the Community via Mailing Lists or [create an issue](https://docs.github.com/es/issues/tracking-your-work-with-issues/creating-an-issue) yourself! 

Reviewer: Could be close to the help forum. We need to go through this information before asking for help in the Galaxy help forum.

Author: If useful links is kept, it'd should be on top of it then

Reviewer: I would not keep "useful links". The goal of this starter-kit is to guide a newcomer step by step through resources that are most useful when you need them.  

Once you have been through a few tutorials, visite the glossary (and you probably come back very often to check definitions), understood you can use main Galaxy servers for starting your Galaxy journey, understood the Galaxy community (and its regional and scientific communities) and understood that you have you are welcome and have a place (with eventually the possibilty to create your own new community), then it is time to take advantage of the Galaxy website itself. So maybe as a last step, I would briefly explain how the Galaxy website is organized to help a newbee to navigate this jungle. all the resources mentioned in these useful links are easily reachable from the website menus.

# Welcome!

<span style="font-size:medium;">This page will be your starting point for Galaxy, the go-to place when you need to refresh your knowledge once you have already get to know more about the Community.</span>

<span style="font-size:medium;">Below you will find a __step-by-step suggested pathway__ for you to follow as a newcomer, a student, an expert or just about anybody interested in using Galaxy.</span>

# Step 1: Get acquainted with Galaxy terminology

The [__Galaxy Glossary__ <i class="fa fa-solid fa-book" aria-hidden="true"></i>](/get-started/glossary/)  contains the terminology, acronyms and definitions. As a newbie you will probably skim through this list, but later on in your journey you may come back to refer to it as it will help you get a better understanding of the Galaxy Ecosystem. 

# Step 2 : Get familiar with Galaxy

<span style="font-size:large;">**Public Servers**</span>

The easiest way to be able to follow our tutorials and learn first-hand about Galaxy is by using a[ __Public Galaxy Server__](/use/). We recommend using one of these :

> - [**Main Server**](https://usegalaxy.org)

Author: This should also be change in the public instances under usegalaxy as well? It sais Main Server there as well

|How to create a new tutorial in the GTN   |   |[<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial/tutorial.html){:target="_blank"}   |[<i class="fas fa-video" aria-hidden="true"></i>](https://www.youtube.com/watch?v=Vw9h5kvRH2k){:target="_blank"}  |   |   |

## **Events**

Before diving into Galaxy tutorials, you may check [**Galaxy events**](/events/) and see if there is any upcoming Galaxy Training events you could join. It is often easier and fun to learn together!

## **Galaxy Training Network**

We have many more Galaxy Training materials [**here**](https://training.galaxyproject.org/)! Galaxy Training Network has a great list of tutorials from a variety of topics (_Climate, Transcriptomics, Data Science ,Development and more!_) written and maintained by the Worldwide Galaxy Community.

## **Galaxy Help & Bug Reports**

<img class="img-fluid float-right" src="/images/undraw-illustrations/fixing-bugs.svg" style="width:200px;height:100px;" alt="community events" />

> Have a bug  <i style ="font-size:small;" class="fa fa-solid fa-bug"></i> or an error you haven't been able to solve? Do you have a question that you haven't found already answered in the FAQs? 

Reviewer: Maybe we could have the link to the FAQs here.

Author: Done

# Step 3:  Your place within the Galaxy Community

<img class="img-fluid float-right" src="/images/undraw-illustrations/connected-people.svg" style="width:200px;height:100px;" alt="community events" />

Still wonder what the Galaxy Community is about?
You already know it, you are already part of it! But let’s dive in the complexity of the [Galaxy Communities](/community/)!

## **Regional Galaxy Communities**
From India to Australia, Japan to France and many more countries [**Regional Communities**](/community/#regional-communities) are a big part of Galaxy. Find ways to contact them, platforms and lots of other useful links.

## **Scientific Galaxy Communities**
Another way communities are formed is by domains such as Proteomics,Transcriptomics,Ecology and Genome annotation to name a few. Most have set up their own servers and mailing list, look for them [**here**](/community/#regional-communities)

## **Set up your own Community**
What if you can't find yourself in these lists of regional and/or scientific communities? Galaxy welcomes new communities, please see [**here**](/get-started/new-leads).

# Step 4: Take part and contribute to Galaxy

> [Here's](https://galaxyproject.org/community/contributing/) our guide on how to start contributing and there are many ways to do so: coding,writing and creating tutorials, answering questions in Gitter or the Help Forum and many more.

Reviewer: a space is missing between coding,writing --> coding, writing

I would also add a link to gitter and agin the the help forum.

## **Working Groups**
Interested on a particular area within Galaxy? Connect with a [**Working Group**](community/wg/). Find the repository, Gitter channel and GDrive for each one.

 ## **Galaxy Mentorship Network**,

Are you ready & excited to start your own project or community, to contribute to Open source or to settle into the Galaxy ecosystem?
Check out our [**mentoring program**](#) 

[#]:I cannot find the link

> 

Reviewer: yes, it think they haven't "re-activated" the mentoring program website!

### Galaxy Help & Bug Reports

<img class="img-fluid float-right" src="/images/undraw-illustrations/fixing-bugs.svg" style="width:200px;height:100px;" alt="community events" />

Have a bug  <i style ="font-size:small;" class="fa fa-solid fa-bug"></i> or an error you haven't been able to solve? Do you have a question that you haven't found already answered in the [**FAQs**](https://training.galaxyproject.org/training-material/faq)? 

Check our [**issues dedicated page**](/issues/) for known issues, problems and tips. Galaxy uses [**GitHub issues**](https://github.com/galaxyproject/galaxy/issues) for tracking down bugs and feature requests, **please make sure you search the list for duplicates before creating a new issue**.

If you couldn't find a solution, reach out to the [**Galaxy Help Forum**](https://help.galaxyproject.org/) as *the* place to ask your questions related to Galaxy use or  [**create an issue**](https://docs.github.com/es/issues/tracking-your-work-with-issues/creating-an-issue) yourself. Either way, we are more than happy to help you out!

### Mailing lists

> If you haven't used mailing list before they are simply a list of addresses to which the same information is being sent, just another way to communicate and interact with the Community. See them [**here**](/mailing-lists/) and remember [**don't be afraid to ask questions**](/mailing-lists#new-to-mailing-lists/)

Reviewer: dot is missing at the end of the sentence.

and remember [**don't be afraid to ask questions**](/mailing-lists#new-to-mailing-lists/).

### Galaxy Publication Library
Check what others members have done in our [**Publication library**](https://www.zotero.org/groups/1732893/galaxy) which contains publications that use, reference, implement, and extend Galaxy.

## Step 3:  Your place within the Galaxy Community

<img class="img-fluid float-right" src="/images/undraw-illustrations/connected-people.svg" style="width:200px;height:100px;" alt="community events" />

Still wonder what the Galaxy Community is about?
You already know it, you are already part of it! But let’s dive in the complexity of the [Galaxy Communities](/community/)!

### Regional Galaxy Communities
From India to Australia, Japan to France and many more countries [**Regional Communities**](/community/#regional-communities) are a big part of Galaxy. Find ways to contact them, platforms and lots of other useful links.

> ### Galaxy Communities of Practice Communities

Reviewer: I think we have too many times the word "communities" in the title. Maybe 

### Galaxy Communities of practice

> Another way communities are formed is by Community such as Proteomics,Transcriptomics,Ecology and Genome annotation to name a few. Most have set up their own servers and mailing list, look for them [**here**](/community/#communities-of-practice)

Reviewer: by domain instead of by Community e.g.

Another way communities are formed is by domain such as Proteomics,Transcriptomics,Ecology and Genome annotation to name a few.

Reviewer: And add a dot at the end of the sentence e.g. 

look for them [**here**](/community/#communities-of-practice).

### Set up your own Community

> What if you can't find yourself in these lists of regional and/or community of practice? Galaxy welcomes new communities, please see [**here**](/get-started/new-leads).

Reviewer 2: ```suggestion
What if you can't find your interests in these lists of regional and/or community of practice? Galaxy welcomes new communities, please see [**here**](/get-started/new-leads).
```

What if you can't find yourself in these lists of regional and/or community of practice? Galaxy welcomes new communities, please see [**here**](/get-started/new-leads).

## Step 4: Take part and contribute to Galaxy

[Here's](https://galaxyproject.org/community/contributing/) our guide on how to start contributing and there are many ways to do so: coding, writing and creating tutorials, answering questions in Gitter or the Help Forum and many more.

### Working Groups

> Interested on a particular area within Galaxy? Connect with a [**Working Group**](/community/wg/). Find the repository, Gitter channel and GDrive for each one.

Reviewer: I don't understand the sentence:

Find the repository, Gitter channel and GDrive for each one.

- This is more about the Github Project where they have some kind of "roadmap" or at least info on they past, current and future work (and not necessarily about the github repository)
- GDrive: I would put Google Drive; in case someone wonders about this "acronym".

> ### Galaxy Mentorship Network

Reviewer 2: ```suggestion
### Galaxy Mentoring Network
```

Reviewer 2: I think I made this suggestion already? Am I getting confused with the version?

Author: I've wrote the same in another file, you're not confused

> Are you ready and excited to start your own project or community, to contribute to Open source or to settle into the Galaxy ecosystem?

Reviewer 2: ```suggestion
Are you ready and excited to start your own project or community, to contribute to Open Source or to settle into the Galaxy ecosystem?
```

> Check out our [**mentoring program.**](https://galaxy-mentor-network.netlify.app/) 

Reviewer 2: ```suggestion
Check out the [**Galaxy mentoring program.**](https://galaxy-mentor-network.netlify.app/).

```
